src/main/java/ca/bc/gov/chefs/etl/forms/pcd/hrRecords/model/HRRecordsData.java

```diff
@@ -0,0 +1,236 @@
+    public String fteEquivalent;
+    public String paymentModality;
+    public String dateHired;
+    public String fiscalYear;
+    public String period;
+    public String employmentStatus;
+    public String dateEmploymentStatusChanged;
+    public String recordCreationDate;
+    public String notes;
+    public String legacyWebformId;
+
+    public String getSubmissionId() {
+        return submissionId;
+    }
+    public void setSubmissionId(String submissionId) {
+        this.submissionId = submissionId;
+    }
+    public String getHrRecordId() {
+        return hrRecordId;
+    }
+    public void setHrRecordId(String hrRecordId) {
+        this.hrRecordId = hrRecordId;
+    }
+    public String getPractitionerType() {
+        return practitionerType;
+    }
+    public void setPractitionerType(String practitionerType) {
+        this.practitionerType = practitionerType;
+    }
+    public String getPractitionerName() {
+        return practitionerName;
+    }
+    public void setPractitionerName(String practitionerName) {
+        this.practitionerName = practitionerName;
+    }
+    public String getPractitionerFirstName() {
+        return practitionerFirstName;
+    }
+    public void setPractitionerFirstName(String practitionerFirstName) {
+        this.practitionerFirstName = practitionerFirstName;
+    }
+    public String getPractitionerLastName() {
+        return practitionerLastName;
+    }
+    public void setPractitionerLastName(String practitionerLastName) {
+        this.practitionerLastName = practitionerLastName;
+    }
+    public String getPractitionerRole() {
+        return practitionerRole;
+    }
+    public void setPractitionerRole(String practitionerRole) {
+        this.practitionerRole = practitionerRole;
+    }
+    public String getPractitionerBillingNumber() {
+        return practitionerBillingNumber;
+    }
+    public void setPractitionerBillingNumber(String practitionerBillingNumber) {
+        this.practitionerBillingNumber = practitionerBillingNumber;
+    }
+    public String getPractitionerBillingNumberNotAvailable() {
+        return practitionerBillingNumberNotAvailable;
+    }
+    public void setPractitionerBillingNumberNotAvailable(String practitionerBillingNumberNotAvailable) {
+        this.practitionerBillingNumberNotAvailable = practitionerBillingNumberNotAvailable;
+    }
+    public String getSpecialty() {
+        return specialty;
+    }
+    public void setSpecialty(String specialty) {
+        this.specialty = specialty;
+    }
+    public String getOtherSpecialty() {
+        return otherSpecialty;
+    }
+    public void setOtherSpecialty(String otherSpecialty) {
+        this.otherSpecialty = otherSpecialty;
+    }
+    public String getGroupRole() {
+        return groupRole;
+    }
+    public void setGroupRole(String groupRole) {
+        this.groupRole = groupRole;
+    }
+    public String getAdditionalGroupDetails() {
+        return additionalGroupDetails;
```

Comment:
For any fields that come from text areas you need to have code like the following:

![image](https://github.com/bcgov/medis-etl-service/assets/80127167/664379ed-ed3c-4b30-8241-e6eac8a59dbf)